### ISSUE TITLE
tailored -> custom ?

### ISSUE THREAD

**Issue by yarikoptic** [datalad/datalad#3064]
"Inspired" by #3061 I wished there was consistency and we used a single term (e.g. `tailored`) instead of two (`custom` and `tailored`) interchangeably.
But too late now probably to change custom to tailored, and easiest would be to change `tailored` to `custom` in the available values for `-f`?

**Comment by mih** [comment]
I don't think that any of the two prefixes are actually needed. We can simply have a result renderer that comes with the command class (i.e. strip `custom_` from the function names). All we actually need is a label that is used for `--output-format` (ATM this is `tailored`). I would counter-propose to keep that as `tailored` to avoid needless change visible on the outside.
